src/transxchange/read.rs

```diff
@@ -1680,7 +1721,7 @@ mod tests {
                 </child>
             </root>"#;
             let root: Element = xml.parse().unwrap();
-            calculate_stop_times(&stop_points, &root, Time::new(0, 0, 0)).unwrap();
+            calculate_stop_times(&stop_points, &root, Time::new(0, 0, 0), &Vec::new()).unwrap();
```

Comment:
It'd be nice to have a test when the `Vec` is not empty (when there is an actual override of the `JourneyPatternTimingLink`).

Reply:
That's why it's in "do not merge"  :wink: